features/net/FEATURE_IPV4/lwip-interface/.mbedignore

```diff
@@ -0,0 +1,6 @@
+lwip/doc/*
+lwip/test/*
+lwip/src/apps/*
+lwip/src/netif/ppp/*
+lwip/src/inlude/lwip/apps/*
+lwip/src/inlude/posix/*
```

Comment:
Looks like you're right, removing the directories would be annoying.

It is possible to amend the subtree add commit to remove the directories, and this removal works cleanly with future subtree merges. But if the removed files are modified, it will introduce a merge conflict for every modification.